tests/mentorship_relation/test_api_send_request.py

```diff
@@ -14,6 +15,12 @@
 class TestSendRequestApi(MentorshipRelationBaseTestCase):
     def setUp(self):
         super(TestSendRequestApi, self).setUp()
+	self.notes_example = 'description of a good mentorship relation'
```

Comment:
I think this is not well indented and that is why TravisCI is failing

Reply:
I indented it properly, replaced tab with proper spaces.